_layouts/post.html

```diff
@@ -0,0 +1,54 @@
+---
+layout: default
+custom_js:
+- prism
+- guides
+---
+
+<div class="main">
+  <div class="section section-hero">
+    <div class="container">
+      <div class="row">
+        <div class="col-xs-12 text-center">
+          <h1>{{ page.title }}</h1>
+          <h6 class="lead">
+            10 mins read<strong>.</strong> Last updated {{ page.date | date_to_string }}
```

Comment:
We should calculate the read time automatically based on the length of the post.

Reply:
Yes I've fixed this